test/models/rhsm_fact_parser_test.rb

```diff
@@ -4,12 +4,15 @@ module Katello
   class RhsmFactParserTest < ActiveSupport::TestCase
     def setup
       @facts = {
+        'net.interface.bond0.mac_address' => '52:54:00:6D:40:72',
         'net.interface.eth0.mac_address' => '00:00:00:00:00:12',
         'net.interface.eth0.ipv4_address' => '192.168.0.1',
         'net.interface.eth0.1.mac_address' => '00:00:00:00:00:12',
         'net.interface.eth0.1.ipv4_address' => '192.168.0.2',
+        'net.interface.eth1.permanent_mac_address' => '52:54:00:6D:40:72',
```

Comment:
`net.interface.eth1.permanent_mac_address` isn't being used by any of the tests yet.

Reply:
@ianballou I'm on the fence about including this.. my idea was that the reader of the code could see that it's a bonded interface setup, with `bond0` being a joining of `eth1` and `eth2` and each interface showing a `permanent_mac_address` value.  So even though that specific value isn't included in a test, it still shows the theoretical setup of the system that we're testing against.

Reply:
oh I see what you mean, from that perspective that seems fine.  I mostly wanted to make sure it wasn't supposed to be used.

Reply:
Do we have unit tests for both with and without bonded interfaces? The way the facts and tests below are changed make me doubt (but I didn't look through the whole test file to be honest!)

Reply:
@jturel looks like the [test here](https://github.com/Katello/katello/pull/8395/files#diff-55054d7748020a4af5b40484c02acbb2R34) should cover that case.

Reply:
ACK I think you're right!

Reply:
@jturel `test_get_facts_for_interface_without_ip` and `test_get_facts_for_interface_with_ip` were the tests that were there originally.  They weren't written with bonded interfaces in mind as far as I know.  

The new test `test_get_facts_for_bonded_interface` asserts that `permanent_mac_address` and not `mac_address` is used for a _bonded_ interface.  However, I don't think we yet have a test specifically asserting the opposite, e.g. that `mac_address` is the value used for a _non-bonded_ interface.

Reply:
^^ @ianballou good catch, So that test above already uses the `mac_address` value.  I think that should be okay.